### ISSUE TITLE
Access for new contributors

### ISSUE THREAD

**Issue by vitorprado** [bemyeyes/bemyeyes-android#10]
How can I help? I know I can send a pull-request, but what to do? 
Would be awesome if you provide a feature list or something to guide outside contributions.

Thanks!

**Comment by epiclapser** [comment]
Yeah I had the same issue with this whole thing, I tried to add a bit of new code that'd at least put two buttons into it, but no ones accepted it...it'd help if we had some communication going on with the guy who started this.....

**Comment by rslinford** [comment]
We are here. We are brave :-) We can do hard things. We desire a quest worthy of our inflated egos. Or just a simple bug fix request would do.
